### PR TITLE
Finish 'Scorecards' section

### PR DESCRIPTION
**Pull request by dobromirmontauk**
This PR cleans up the expectations/scorecard section and makes it actually useful.

### CODE REVIEW

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:36]: expert of? maybe on or in?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:45]: I'd replace s/he with they

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:51]: " you get incorrectly blocked by external factors." - what does being correctly blocked mean? Maybe just remove "incorrectly"?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:52]: I wonder if in the escalations case we should not change to "you respectfuly escalate..."?
There's a right and the wrong way of doing it, just hinting at it;)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:59]: This seems really low down the list... I wonder if it should not be at least mentioned somewhere above as well, where the impressive deliveries on tight deadlines live...

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:67]: feature -> warrior?

**Review by joannakrk — COMMENTED** [review]
Hey:) took a pass - feel free to incorporate or ignore.
For the scorecards, if accepted, some comments will need to be applied across all of them - I've only put them where I saw them first.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md:65]: Cleaned up by moving to README.md

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md]: Yeah, this is a historical one as well.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md:34]: Oh that was a Twitter-specific organization.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md:36]: This is a historical scorecard that I imported from Twitter so I kind of don't want to change it.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_billing_payments_em.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_billing_payments_em.md:43]: Specifically billing & payments.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md:25]: Agreed. How could we make this scorecard not a 'architecture astronaut' that isn't practical?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md]: Good call out, added.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md]: Thanks, moved to the README.md and fixed the link.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:29]: Yes... by 'low latency' I mean that from request -> delivery of the feature/bug/whatever is very fast. So low on process, tech debt is OK, etc etc. How can I clean this up?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed, thanks.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: I'll change it to 'unnecessarily'. Sometimes you get blocked and it's necessary, e.g. hardware doesn't show up on time b/c of a delay in the ports...

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Oh I love that, fixed.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:55]: Nitpicking :)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed!

**Review by dobromirmontauk — COMMENTED** [review]
THANK YOU for the wonderful comments. I made most of the changes.

**Comment by joannakrk** [comment]
LGTM:) Is closing of all the comments required before you can merge?

**Comment by dobromirmontauk** [comment]
> LGTM:) Is closing of all the comments required before you can merge?

I currently have it set that all conversations need to be resolved + an approval before merging. Mind sending me approval? :)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md:30]: I think maybe it's ok for now... will think about it..

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md:25]: I think I'd just add a pragmatism, as the ability to balance business needs with the inner desire for ultimately beautful generic designs ;)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:29]: You optimize for low-latency process.
or
You optimize for quick turn-around at each step of the process?

**Review by joannakrk — APPROVED** [review]
I hope you can see comments...

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md]: good point. Somehow I assumed... ;)

**Pull request merged**
